src/crypto/local.js

```diff
@@ -373,6 +394,43 @@ function getSeedPRNG(chkM, chkZ, pointG, pointH) {
     return sjcl.codec.hex.fromBits(h.finalize());
 }
 
+/**
+ * hkdf - The HMAC-based Key Derivation Function
+ * based on https://github.com/mozilla/node-hkdf
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * @param {bitArray} ikm Initial keying material
+ * @param {integer} length Length of the derived key in bytes
+ * @param {bitArray} info Key derivation data
+ * @param {bitArray} salt Salt
+ * @param {sjcl.hash} hash hash function
+ * @return {bitArray}
+ */
+function evaluateHkdf(ikm, length, info, salt, hash) {
+    const mac = new sjcl.misc.hmac(salt, hash);
+    mac.update(ikm);
+    const prk = mac.digest();
+
+    const hashLength = sjcl.bitArray.bitLength(prk)/8;
```

Comment:
if ``sjcl.bitArray.bitLength(prk)`` can be < 8 then this is a bug

should be "(sjcl.bitArray.bitLength(prk)+7)/8"

Reply:
fixed